lib/experimental/class-wp-nav-menu-conversion-gutenberg.php

```diff
@@ -0,0 +1,128 @@
+<?php
+/**
+ * WP_Nav_Menu_Conversion_Gutenberg class
+ *
+ * @package gutenberg
+ * @since 6.3.0
+ */
+
+/**
+ * Manages Fallback behavior for Navigation menus.
```

Comment:
```suggestion
 * Manages Fallback behavior for Navigation block menus.
```

Reply:
Looks like the comment is not whay the class name says, probably a remnant. This class converts a classic menu to a block menu.